### PR TITLE
Update backtesting tutorial

### PR DESCRIPTION
**Pull request by isahers1**

### CODE REVIEW

File: docs/evaluation/tutorials/backtesting.mdx [hunk 1]
```diff
@@ -4,118 +4,169 @@ sidebar_position: 5
 
 # Backtesting
 
-Deploying your app into production is just one step in a longer journey continuous improvement. You'll likely want to develop other candidate systems that improve on your production model using improved prompts, llms, indexing strategies, and other techniques. While you may have a set of offline datasets already created by this point, it's often useful to compare system performance on more recent production data.
+Deploying your application is just the beginning of a continuous improvement process.
+Typically, you'll want to refine your system by enhancing prompts, language models, tools, and architectures.
+While the initial datasets you created before deployment are useful for testing new versions, you now have access to real historical data. This data provides a more accurate representation of the inputs your application will encounter in the future.
```
Comment (tanushree-sharma) [inline docs/evaluation/tutorials/backtesting.mdx:9]:
```suggestion
```

File: docs/evaluation/tutorials/backtesting.mdx [hunk 2]
```diff
@@ -4,118 +4,169 @@ sidebar_position: 5
...
+Backtesting involves assessing new versions of your application using this historical data and comparing the new outputs to the original ones.
```
Comment (tanushree-sharma) [inline docs/evaluation/tutorials/backtesting.mdx:10]:
```suggestion
Backtesting involves assessing new versions of your application using historical data and comparing the new outputs to the original ones.
```

File: docs/evaluation/tutorials/backtesting.mdx [hunk 3]
```diff
@@ -4,118 +4,169 @@ sidebar_position: 5
...
+This approach can offer a clearer indication of whether the new version is an improvement over the current deployment compared to evaluations using pre-production datasets.
```
Comment (tanushree-sharma) [inline docs/evaluation/tutorials/backtesting.mdx:11]:
```suggestion
Compared to evaluations using pre-production datasets, backtesting offers a clearer indication of whether the new version of your application is an improvement over the current deployment. 
```

File: docs/evaluation/tutorials/backtesting.mdx [hunk 4]
```diff
@@ -4,118 +4,169 @@ sidebar_position: 5
 
 # Backtesting
 
-Deploying your app into production is just one step in a longer journey continuous improvement. You'll likely want to develop other candidate systems that improve on your production model using improved prompts, llms, indexing strategies, and other techniques. While you may have a set of offline datasets already created by this point, it's often useful to compare system performance on more recent production data.
+Deploying your application is just the beginning of a continuous improvement process.
+Typically, you'll want to refine your system by enhancing prompts, language models, tools, and architectures.
```
Comment (tanushree-sharma) [inline docs/evaluation/tutorials/backtesting.mdx:8]:
```suggestion
After you deploy to production, you'll want to refine your system by enhancing prompts, language models, tools, and architectures.
```

File: docs/evaluation/tutorials/backtesting.mdx [hunk 5]
```diff
@@ -4,118 +4,169 @@ sidebar_position: 5
...
+:::
 
-![](./static/dataset_page.png)
+## Setup
 
-**Note:** In most cases, you won't have "ground truth" answers in this case, but you can manually compare and label or use reference-free evaluators to score the outputs.(If your application DOES permit capturing ground-truth labels, then we obviously recommend you use those.
+### Configure the environment
 
-## Prerequisites
+Install and set environment variables. This guide requires `langsmith>=0.2.4`.
```
Comment (tanushree-sharma) [inline docs/evaluation/tutorials/backtesting.mdx:31]: `This guide requires `langsmith>=0.2.4`.` does this only apply if users are self-hosting?

File: docs/evaluation/tutorials/backtesting.mdx [hunk 6]
```diff
@@ -4,118 +4,169 @@ sidebar_position: 5
...
-2. Convert runs to dataset + initial experiment.
-3. Run new system against the dataset to compare.
+This process will provide you with a new dataset of representative inputs, which you can version and use for backtesting your models.
 
-You will then have a new dataset of representative inputs you can you can version and backtest your models against.
+:::info Ground truth data
+Often, you won't have definitive "ground truth" answers available.
+In such cases, you can manually label the outputs or use evaluators that don't rely on reference data.
+If your application allows for capturing ground-truth labels, for example by allowing users to leave feedback, we strongly recommend doing so.
+:::
```
Comment (tanushree-sharma) [inline docs/evaluation/tutorials/backtesting.mdx:26]: Add a sentence here about what the tutorial covers. eg In this tutorial, we will go over XYZ..

File: docs/evaluation/tutorials/backtesting.mdx (hunk 5 above)
Comment (baskaryan) [reply docs/evaluation/tutorials/backtesting.mdx:30]: no its for all, we use some sdk methods in the code snippets that are only available in that version

**Pull request merged**